src/EnergyPlus/HVACManager.cc

```diff
@@ -3072,7 +2922,7 @@ void SetHeatToReturnAirFlag(EnergyPlusData &state)
 void UpdateZoneInletConvergenceLog(EnergyPlusData &state)
 {
 
-    std::array<Real64, DataConvergParams::ConvergLogStackDepth> tmpRealARR = {0};
+    std::array<Real64, DataConvergParams::ConvergLogStackDepth> tmpRealARR;
```

Comment:
[src/EnergyPlus/HVACManager.cc:3075]:(style),[unreadVariable],Variable 'tmpRealARR' is assigned a value that is never used.